### PR TITLE
Required Resources in React

### PR DESCRIPTION
**Pull request by arutr**
Includes all required resources pages, refactored components to use React Bootstrap package, and a 404 page!

### CODE REVIEW

File: client/src/components/Resource/ForecastResourceSearchResults.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    }
+
+    componentWillReceiveProps(nextProps) {
+        this.setState({
+            results: nextProps.searchResults
+        });
+    }
+
+    render() {
+        // let resultRows = this.state.results.map(function (resource) {
```
Comment (gwolverson) [inline client/src/components/Resource/ForecastResourceSearchResults.js:40]: Are these commented out for a reason?

File: client/src/components/Resource/AddTeamResourceSearchResults.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    }
+
+    componentWillReceiveProps(nextProps) {
+        this.setState({
+            results: nextProps.searchResults
+        });
+    }
+
+    render() {
+        // let resultRows = this.state.results.map(function (resource) {
```
Comment (gwolverson) [inline client/src/components/Resource/AddTeamResourceSearchResults.js:41]: Are these commented out for a reason?

**Review by gwolverson — COMMENTED** [review]
Changes look good, only a couple of comments around code which has been commented out.

**Comment by arutr** [comment]
These have been commented out as search was not working. I am on my way to fixing this on `react_refactor_artur`

**Pull request merged**